kafka-plugins-0.10/widgets/Kafka-batchsink.json

```diff
@@ -13,11 +41,11 @@
           "name": "referenceName"
         },
         {
-          "widget-type": "csv",
-          "label": "Kafka Brokers",
-          "name": "brokers",
```

Comment:
This is backward incompatible change, check my previous comment on how we should properly handle it.

Reply:
done